### ISSUE TITLE
Can't create an One to Many relation with Admin Users

### ISSUE THREAD

**Issue by FelipeMarcos** [strapi/strapi#12195]
### Describe the bug

TL;DR: Blank screen on Edit entry screen.

I have a very simple Collection with a title and a relation with Admin Users, whenever I try to edit an entry, an empty screen is shown with errors in the console.

### Steps to reproduce the behavior

1. Create a collection
2. Add a relation field with User (from: admin)
3. Set it to "has many"
4. Save
5. Create a new Entry and associate a user with it
6. You will immediately see the error once you save

### Expected behavior

Should not show an empty screen when trying to edit an entry.

### Screenshots
<img width="707" alt="Screen Shot 2022-01-14 at 00 30 41" src="https://user-images.githubusercontent.com/3706755/149446896-5a2ccbdc-b7e6-40a1-ae6b-bdebffba3746.png">

<img width="876" alt="Screen Shot 2022-01-14 at 00 31 33" src="https://user-images.githubusercontent.com/3706755/149447257-9c897ad6-123e-4ae9-a248-3c91c6c54694.png">

<img width="313" alt="Screen Shot 2022-01-14 at 00 36 21" src="https://user-images.githubusercontent.com/3706755/149447349-174b3f38-3eee-44b7-900d-cba180521861.png">

### Code snippets

```js
{
  "kind": "collectionType",
  "collectionName": "sites",
  "info": {
    "singularName": "site",
    "pluralName": "sites",
    "displayName": "Site",
    "description": ""
  },
  "options": {
    "draftAndPublish": true
  },
  "pluginOptions": {},
  "attributes": {
    "title": {
      "type": "string"
    },
    "admin_users": {
      "type": "relation",
      "relation": "oneToMany",
      "target": "admin::user"
    }
  }
}
```

### System

- Node.js version: v14.17.6
- NPM version: 8.1.4
- Strapi version: v4
- Database: MySQL
- Operating system: macOS

**Comment by derrickmehaffy** [comment]
Should be fixed in v4.0.5 as it's a duplicate of https://github.com/strapi/strapi/issues/11891
